src/main/java/uk/gov/pay/directdebit/mandate/model/MandateState.java

```diff
@@ -3,21 +3,22 @@
 import uk.gov.pay.directdebit.mandate.api.ExternalMandateState;
 import uk.gov.pay.directdebit.payments.model.DirectDebitState;
 
+import static uk.gov.pay.directdebit.mandate.api.ExternalMandateState.ABANDONED;
 import static uk.gov.pay.directdebit.mandate.api.ExternalMandateState.INACTIVE;
+import static uk.gov.pay.directdebit.mandate.api.ExternalMandateState.PENDING;
 import static uk.gov.pay.directdebit.mandate.api.ExternalMandateState.STARTED;
 
 public enum MandateState implements DirectDebitState {
     CREATED(ExternalMandateState.CREATED),
     AWAITING_DIRECT_DEBIT_DETAILS(STARTED),
-    USER_CANCEL_NOT_ELIGIBLE(ExternalMandateState.CANCELLED),
-    SUBMITTED_TO_PROVIDER(ExternalMandateState.PENDING),
-    SUBMITTED_TO_BANK(ExternalMandateState.PENDING),
+    SUBMITTED_TO_PROVIDER(PENDING),
+    SUBMITTED_TO_BANK(PENDING),
     ACTIVE(ExternalMandateState.ACTIVE),
-    FAILED(INACTIVE),
+    FAILED(ExternalMandateState.FAILED),
```

Comment:
Since we’ve going to have to use the long form of the name for this one, we may as well for the others for consistency.

Reply:
for payments we've called them EXTERNAL_XXX, is that preferable?

Reply:
Yes, let’s do that